### PR TITLE
postgresql as dep in charts

### PR DESCRIPTION
**Pull request by fgeorgatos**
factorize charts deps - the 1st blood

addresses #208

### CODE REVIEW

File: charts/renku/templates/configmap.yaml [hunk 1]
```diff
@@ -63,3 +63,84 @@ data:
...
+        *.sql)    echo "$0: running $f"; "${psql[@]}" -f "$f"; echo ;;
+        *.sql.gz) echo "$0: running $f"; gunzip -c "$f" | "${psql[@]}"; echo ;;
+        *)        echo "$0: ignoring $f" ;;
+      esac
+      echo
+    done
+
+  init-keycloak-db.sh: |-
+    #!/bin/bash
+    set -e
```
Comment (leafty) [inline charts/renku/templates/configmap.yaml]: TODO: `set -ex`

File: charts/renku/templates/post-install-job.yaml [hunk 2]
```diff
@@ -8,12 +8,12 @@ metadata:
...
   annotations:
     "helm.sh/hook": post-install,post-upgrade
-    "helm.sh/hook-weight": "5"
+    "helm.sh/hook-weight": "20"
     "helm.sh/hook-delete-policy": "before-hook-creation"
 spec:
   template:
     metadata:
-      name: "{{.Release.Name}}"
+      name: "{{.Release.Name}}-post-install"
```
Comment (leafty) [inline charts/renku/templates/post-install-job.yaml]: TODO: rename to `{{ ... }}-post-install-gitlab`

**Comment by fgeorgatos** [comment]
@leafty : re-review?

File: charts/renku/templates/post-install-job.yaml [hunk 3]
```diff
@@ -8,12 +8,12 @@ metadata:
...
   annotations:
     "helm.sh/hook": post-install,post-upgrade
-    "helm.sh/hook-weight": "5"
+    "helm.sh/hook-weight": "20"
     "helm.sh/hook-delete-policy": "before-hook-creation"
 spec:
   template:
     metadata:
-      name: "{{.Release.Name}}"
+      name: "{{.Release.Name}}-post-install-gitlab"
```
Comment (leafty) [inline charts/renku/templates/post-install-job.yaml]: make it match with the name above

File: charts/renku/templates/configmap.yaml [hunk 4]
```diff
@@ -63,3 +64,84 @@ data:
     # configure the logout redirect
     curl -f -is -X PUT -H "Private-token: ${GITLAB_SUDO_TOKEN}" \
       ${GITLAB_SERVICE_URL}/api/v4/application/settings?after_sign_out_path={{ template "http" . }}://{{ .Values.global.renku.domain }}/auth/realms/Renku/protocol/openid-connect/logout?redirect_uri=${GITLAB_URL}
+
+  # Init scripts that populate /docker-entrypoint-initdb.d
+
+  # Unashamedly copied from: https://github.com/docker-library/postgres/blob/master/9.6/docker-entrypoint.sh
+  init-postgres.sh: |-
```
Comment (rokroskar) [inline charts/renku/templates/configmap.yaml]: This is going to keep crashing and recreating new pods to run the job until the server is available -- a better solution might be to wait for the server to be up before continuing. See the other post-install script: https://github.com/SwissDataScienceCenter/renku/pull/293/files#diff-0f15bad1c1882fee30ebc792c5a38350R24

Comment (leafty) [reply charts/renku/templates/configmap.yaml]: That job is doing a curl to wait. Would need an equivalent psql command to ping.

Comment (rokroskar) [reply charts/renku/templates/configmap.yaml]: yes I wasn't suggesting that what the other script does is ideal ;)  
But IMO the script should do the waiting, not rely on k8s to just keep relaunching the pod.

Comment (rokroskar) [reply charts/renku/templates/configmap.yaml]: You could also install curl in the postgres container?

**Comment by rokroskar** [comment]
Please cleanup the commits (either squash all at the end or selectively by hand and force-push)

File: charts/renku/templates/configmap.yaml (hunk 4 above)
Comment (fgeorgatos) [reply charts/renku/templates/configmap.yaml]: @rokroskar , @leafty : eh, what's the action point here? r we settled on a curl trick?

Comment (rokroskar) [reply charts/renku/templates/configmap.yaml]: @fgeorgatos it was fixed in 864f2a0

**Pull request merged**